examples/Training/javascript/index.html

```diff
@@ -0,0 +1,280 @@
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<!-- Example file for connecting and logging in to the JSON api -->
+
+<html>
+
+<link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+
+<style>
+
+#base_url_container {
+  z-index: 100;
+}
+
+#login_container {
+  display: none;
+}
+
+.full_page {
+  height: 100%;
+  width: 100%;
+  position: absolute;
+  top: 0;
+  z-index: 0;
+  background-color: #eee;
+}
+
+.full_page input {
+  margin-bottom: 20px;
+  padding: 10px;
+  font-size: 16px;
+  height: auto;
+}
+
+.navbar {
+  z-index: 0;
+}
+
+#username {
+  margin-bottom: 5px;
+}
+
+body {
+  background-color: #eee;
+}
+
+#server_form, #login_form {
+  max-width: 430px;
+  padding-top: 55px;
+  margin: 0 auto;
+
+}
+
+</style>
+
+<body>
+
+  <div id="base_url_container" class="full_page container">
+    <form id="server_form">
+      <h2>Enter OMERO.web url</h2>
+      <input id="base_url" type="text" class="form-control" placeholder="e.g. http://localhost:4080/" required autofocus>
+      <button class="btn btn-lg btn-primary btn-block" type="submit">Connect</button>
+    </form>
+  </div>
+
+  <div id="login_container" class="full_page container">
+    <form id="login_form">
+      <h2>Login to OMERO</h2>
+      <select id="server"></select>
+      <label for="username" class="sr-only">Username</label>
+      <input id="username" type="text" class="form-control" name="username" placeholder="Username">
+      <label for="password" class="sr-only">Password</label>
+      <input id="password" type="text" class="form-control" name="password" placeholder="Password">
+      <button class="btn btn-lg btn-primary btn-block" type="submit">Login</button>
+    </form>
+  </div>
+
+
+  <nav class="navbar navbar-inverse navbar-fixed-top">
+    <div class="container">
+      <div class="navbar-header">
+        <a class="navbar-brand" href="#">OMERO</a>
+      </div>
+      <div id="navbar" class="collapse navbar-collapse">
+        <form class="navbar-form navbar-right" >
+          <button id="logout_button" type="submit" class="btn btn-link">Log out</button>
+        </form>
+        <ul class="nav navbar-nav navbar-right">
+          <li><a id="logged_in_user" href="#"></a></li>
+        </ul>
+      </div>
+    </div>
+  </nav>
+
+  <div class="container" style="margin-top: 50px">
+    <div class="starter-template">
+      <h1>Projects</h1>
+      <div id="projects">
+      </div>
+    </div>
+  </div>
+
+<script>
+
+var omeroweb_url;
+var latest_base_url;
+var base_urls;
+var csrf_token;
+
+
+function createCORSRequest(method, url) {
+  var xhr = new XMLHttpRequest();
+  if ("withCredentials" in xhr) {
+    xhr.withCredentials = true;
+    xhr.open(method, url, true);
+  } else if (typeof XDomainRequest != "undefined") {
```

Comment:
I have not been able to test that part of the example since I do not have IE accessible at the moment